numpy/f2py/rules.py

```diff
@@ -561,7 +561,8 @@
                  '\tint #name#_return_value_len = 0;'],
         'callfortran':'#name#_return_value,#name#_return_value_len,',
         'callfortranroutine':['\t#name#_return_value_len = #rlength#;',
-                              '\tif ((#name#_return_value = (string)malloc(sizeof(char)*(#name#_return_value_len+1))) == NULL) {',
+                              '\tif ((#name#_return_value = (string)malloc('
+                              '#name#_return_value_len+1) == NULL) {',
```

Comment:
I think all .py files under `numpy/f2py/` require linting fixes. It would result in a PR with a large changeset and I wonder what would be the best way to do that? For instance, such a PR should have a short lifetime to reduce the chances of conflicts with other PRs.
What do you think? @eric-wieser @mattip

Reply:
I think only @pearu and @melissawm are creating PRs under `numpy/f2py`, so if the two of you want to get some set of PRs in while declaring a moratorium on new ones and then go for a big clean up it might be nice.

Reply:
Sounds good to me